com.github.GradienceTeam.Gradience.json

```diff
@@ -0,0 +1,138 @@
+{
+    "app-id": "com.github.GradienceTeam.Gradience",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "42",
+    "sdk": "org.gnome.Sdk",
+    "command": "gradience",
+    "finish-args": [
+        "--share=network",
```

Comment:
do you really need network?

Reply:
Yes, because there is a kind of marketplace